addons/point_of_sale/static/src/app/screens/product_screen/product_screen.js

```diff
@@ -455,6 +455,10 @@ export class ProductScreen extends Component {
         const info = await reactive(this.pos).getProductInfo(productTemplate, 1);
         this.dialog.add(ProductInfoPopup, { info: info, productTemplate: productTemplate });
     }
+
+    isValidToPay() {
+        return true;
+    }
```

Comment:
You don't need to create this method here. You can do everything by overriding xml in pos_loyalty.